content/_assets/styles/components/accordion.scss

```diff
@@ -0,0 +1,282 @@
+$controls-width: 18px;
+$controls-margin: 10px;
+$copy-button-offset: 6px;
+$copy-button-width: 20px;
+$tooltip-padding: 10px;
+
+// Summary
+.accordion-section {
+  margin: 1rem 0 1rem 0;
+}
+
+// Heading
+.quire-page .accordion-section .accordion-section__heading {
+  & > h1,
+  & > h2,
+  & > h3,
+  & > h4,
+  & > h5,
+  & > h6 {
```

Comment:
We might want to make this `:first-child :is(h2, h3, h4, h5, h6) { ... }`

Note that `h1` is omitted from the selector list because semantically it should probably never be in an accordion.